pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -679,3 +681,11 @@ func recordExtensionPointDuration(start time.Time, extensionPoint string, status
 	}
 	metrics.FrameworkExtensionPointDuration.WithLabelValues(extensionPoint, statusCode).Observe(metrics.SinceInSeconds(start))
 }
+
+func recordPermitWaitDuration(start time.Time, status *Status) {
+	res := Success.String()
+	if status != nil {
+		res = status.Code().String()
+	}
```

Comment:
`res = status.Code().String` is OK because `status.Code()` will return `Success` when status is nil, check it [here](https://github.com/kubernetes/kubernetes/blob/7e53c9d80808e44ae0b320f01fe46a03eda2ccf6/pkg/scheduler/framework/v1alpha1/interface.go#L107).

Reply:
Thanks. Removed nil check.